cypress/support/flightSearch/flightSearchCommands.cy.js

```diff
@@ -0,0 +1,29 @@
+Cypress.Commands.add("selectDeparture", () => {
+    cy.get('#input-button__departure').click()
+    
+    cy.get(':nth-child(2) > :nth-child(1) > .countries__country-inner').click()
+    
+    cy.get(':nth-child(1) > .body-l-lg > span').click()
+})
+
+Cypress.Commands.add("selectDestination", () => {
```

Comment:
you don't necessarily need to put all the actions into commands. 
I would recommend you reading through what is good to put into the commands and what should live in page object models (POMs)